Pod/Classes/SKYRecord.h

```diff
@@ -76,7 +76,7 @@ extern NSString *const SKYRecordTypeUserRecord;
 - (id _Nullable)objectForKeyedSubscript:(id)key;
 
 /// Undocumented
-- (void)setObject:(id)object forKey:(id<NSCopying> _Nullable)key;
+- (void)setObject:(id _Nullable)object forKey:(id<NSCopying> _Nullable)key;
 /// Undocumented
 - (void)setObject:(id)object forKeyedSubscript:(id<NSCopying> _Nullable)key;
```

Comment:
This should also be marked with `_Nullable`.

Reply:
Thanks, updated.